packages/frontend/editor-ui/src/app/components/MainHeader/WorkflowHeaderDraftPublishActions.vue

```diff
@@ -100,6 +100,28 @@ const showPublishIndicator = computed(() => {
 	);
 });
 
+const publishTooltipText = computed(() => {
+	const wfHasAnyChanges =
+		workflowsStore.workflow.versionId !== workflowsStore.workflow.activeVersion?.versionId;
+
+	if (!containsTrigger.value) {
+		return i18n.baseText('workflows.publishModal.noTriggerMessage');
+	}
+
+	if (workflowsStore.nodesIssuesExist) {
+		return i18n.baseText('workflowActivator.showMessage.activeChangedNodesIssuesExistTrue.title', {
+			interpolate: { count: workflowsStore.nodesWithIssues.length },
```

Comment:
<img width="350" height="147" alt="Image" src="https://github.com/user-attachments/assets/2adac7f0-1cd5-420b-8f75-e024cfd0c4cf" />
The tooltip looks a bit off with these messages. Maybe increase the width?